test/e2e/framework/util.go

```diff
@@ -339,10 +339,11 @@ func SkipUnlessProviderIs(supportedProviders ...string) {
 }
 
 func SkipUnlessMultizone(c clientset.Interface) {
-	zones, err := GetClusterZones(c)
+	nodes, err := c.CoreV1().Nodes().List(metav1.ListOptions{})
```

Comment:
Don't need a separate node call; you can check the error from `GetClusterZones` instead.

Reply:
@verult Done